.circleci/config.yml

```diff
@@ -266,7 +268,8 @@ jobs:
       - capture_test_logs
 
   acceptanceTestsQuorum:
-    executor: quorum_ats_executor_med
+    parallelism: 1
```

Comment:
Does this have an effect? Or does it just explicitly turn parallelism off?

Reply:
tbh I'm not 100% sure but I'm reverting back to how it was when this CI was passing. Once it's passing, we can go from there.